src/app/wizard/views/DataplaneUniversal.spec.ts

```diff
@@ -1,27 +1,10 @@
-import { createRouter, createWebHashHistory } from 'vue-router'
 import { rest } from 'msw'
 import { render } from '@testing-library/vue'
 import userEvent from '@testing-library/user-event'
 
 import DataplaneUniversal from './DataplaneUniversal.vue'
 import { store, storeKey } from '@/store/store'
```

Comment:
nitpick: not introduced in this PR but if GH action is right and we can remove it then let's do it (like you did in ZoneIngresses).

Reply:
Cleaned this up.